frontend/src/old-pages/Configure/Queues/queues.mapper.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+// with the License. A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  SingleInstanceComputeResource,
+  MultiInstanceComputeResource,
+} from './queues.types'
+
+function mapComputeResource(
+  computeResource: SingleInstanceComputeResource | MultiInstanceComputeResource,
+): MultiInstanceComputeResource {
+  if ('Instances' in computeResource) {
+    return computeResource
+  }
+
+  const {InstanceType, ...otherComputeResourceConfig} = computeResource
+
+  return {
```

Comment:
issue: this check should be applied **only** when running a PC version >= 3.3.0. For all the other versions, the behavior is unchanged.